dvc/repo/unprotect.py

```diff
@@ -13,13 +13,10 @@
 def _unprotect_file(path):
     if System.is_symlink(path) or System.is_hardlink(path):
         logger.debug("Unprotecting '{}'".format(path))
-
         tmp = os.path.join(os.path.dirname(path), "." + str(uuid.uuid4()))
-        move(path, tmp)
-
-        copyfile(tmp, path)
-
-        remove(tmp)
+        copyfile(path, tmp)
```

Comment:
I think it would be beneficial to put a comment that order is important. That same motivation you summarized in this PR.

Reply:
Done